api/src/server.ts

```diff
@@ -6,13 +6,14 @@ import * as Tracing from "@sentry/tracing";
 import * as notangles from "./notangles/index";
 import * as freerooms from "./freerooms/index";
 import { writeData } from "./write-data";
+import { getStartDate } from "./getStartDate";
 
 const app = express();
 const port = process.env.PORT || 3001;
 
 // Sentry configurations
 Sentry.init({
-    dsn: `${process.env.SENTRY_INGEST_URL}`,
+    dsn: `${process.env.INGEST_URL}`,
```

Comment:
Why was this changed?

Reply:
It wasn't meant to be changed I've asked Raiyan to fix it dw

Reply:
Yep, it was mentioned to me earlier, thanks for the reminder nonetheless!

Reply:
Ah sorry I wasn't aware; all good!